connector/src/main/java/org/geysermc/connector/GeyserConnector.java

```diff
@@ -102,6 +108,10 @@
     @Setter
     private AuthType defaultAuthType;
 
+    private TimeSyncer timeSyncer;
```

Comment:
We can probably make at least this field final (set it to null if not Floodgate).